woocommerce/woocommerce-gutenberg-products-block
# Blocks API - Reviews endpoint with rating sort and category filtering

Updates the Reviews endpoints so they are available for guests. Sensitive data is removed. This duplicates and customises the core endpoint so we can remove unwanted functionality.

Also adds the ability to sort by rating for #658.

Important: I've renamed some collection args to make more sense, and introduced category filtering for #589:

- `product` -> `product_id`
- `category_id`

### How to test the changes in this Pull Request:

1. Request `http://local.wordpress.test/wp-json/wc/blocks/products/reviews` whilst logged out.
    a) confirm approved reviews are listed
    b) confirm no personal data (email address) is shown
2. Request `http://local.wordpress.test/wp-json/wc/blocks/products/reviews?orderby=rating`
    a) confirm reviews are sorted by rating
3. Request `http://local.wordpress.test/wp-json/wc/blocks/products/reviews?category_id=X` where X is a category of products. Check you get reviews only for products in that category.
